### PR TITLE
Replace repetitive actions with Action Groups in CreateProductAttributeEntityWithReservedKeysTest

### LINKED ISSUE #34422: [Issue] Replace repetitive actions with Action Groups in CreateProductAttributeEntityWithReservedKeysTest
This issue is automatically created based on existing pull request: magento/magento2#34419: Replace repetitive actions with Action Groups in CreateProductAttributeEntityWithReservedKeysTest

---------

### Fixed Issues (if relevant)
Tests are refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
Run test locally

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Fixed Issues (if relevant)
Tests are refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
Run test locally

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#34422: Replace repetitive actions with Action Groups in CreateProductAttributeEntityWithReservedKeysTest

### CODE REVIEW

**Review by edenduong — APPROVED** [review]
Thank you for your contribution!

File: app/code/Magento/Catalog/Test/Mftf/Test/CreateProductAttributeEntityTest/CreateProductAttributeEntityWithReservedKeysTest.xml [hunk 1]
```diff
@@ -26,22 +26,28 @@
         </after>
 
         <!--Navigate to Stores > Attributes > Product.-->
-        <amOnPage url="{{AdminProductAttributeGridPage.url}}" stepKey="goToProductAttributesGrid"/>
+        <!--<amOnPage url="{{AdminProductAttributeGridPage.url}}" stepKey="goToProductAttributesGrid"/>-->
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/Test/CreateProductAttributeEntityTest/CreateProductAttributeEntityWithReservedKeysTest.xml]: Please remove the commented `amOnPage` action.
```suggestion
```

**Comment by eduard13** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run all tests

**Review by eduard13 — APPROVED** [review]

**Pull request merged**